brainsets/datasets/AllenVisualCodingOphys2016.py

```
@@ -0,0 +1,48 @@
+        )
+
+        self.split_type = split_type
+
+        # ROI IDs are unique across sessions in this dataset
+        self.calcium_imaging_dataset_mixin_uniquify_roi_ids = False
+
+    def get_sampling_intervals(
+        self,
+        split: Optional[Literal["train", "valid", "test"]] = None,
+    ):
+        if self.split_type == None:
```

Ruff (0.14.14)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)